server/sonar-db-dao/src/main/java/org/sonar/db/user/UserDto.java

```diff
@@ -48,6 +48,8 @@
   private String salt;
   private Long createdAt;
   private Long updatedAt;
+  private String homepageType;
+  private String homepageKey;
```

Comment:
I thought we agreed on homepage_value. We must clarify

Reply:
done